client/src/layout/Search.js

```diff
@@ -1,35 +1,43 @@
 import React, {Component} from 'react';
 import axios from 'axios';
+import ProfilePreview from "../profiles/ProfilePreview";
 
 class Search extends Component {
     state = {
-        text: ''
+        text: '',
+        cats: [],
+        showNoResults: false,
+    };
+
+    searchCats = async text => {
+        const catUrl = `http://localhost:5000/api/cats/search/${text}`;
```

Comment:
At some point move the backend url to an environment variable so it can be different when deployed